azure-kusto-data/source/client.ts

```diff
@@ -128,13 +138,12 @@ export class KustoClient {
                      properties?: ClientRequestProperties | null): Promise<KustoResponseDataSet> {
         const axiosConfig = {
             headers,
-            gzip: true,
```

Comment:
Why remove the gzip: true?

Reply:
I found that it not part from axiosConfigurations so looks like It doesn't do anything and leftover from old client.
IIUC - we get it from the Accept-Encoding header